website/data/cdktf-nav-data.json

```diff
@@ -173,27 +173,132 @@
     "routes": [
       {
         "title": "Overview",
-        "path": "api-reference/overview"
```

Comment:
Can / Should we redirect this to just `api-reference` for old links to still work?